app/perftools/management/commands/create_page_cache.py

```diff
@@ -49,6 +50,208 @@
+            },
+            {
+                'name': 'EPNS',
+                'logo_url': static('v2/images/jtbd/epns.png'),
+                'description': 'EPNS is a decentralized DeFi notifications protocol which enables users (wallet addresses) to receive notifications. Using the protocol, any dApp, smart contract or service can send notifications to users(wallet addresses) in a platform agnostic fashion (mobile, web, or user wallets)',
+            },
+        ],
+        'alumni': alumni,
+        'hackathons': fetch_jtbd_hackathons(),
+        'testimonial': {
+            'handle': 'sebastian',
+            'role': 'Python Developer',
+            'comment': "Transitioning to a career in crypto can be tough, but Gitcoin was a big help for me. Completing Gitcoin bounties and participating in Hackathons were invaluable for gaining exposure, experience, and of course making some money!",
+            'avatar_url': '',
+            'twitter': '',
+            'github': '',
+        },
+    }
+    view = 'jtbd'
+    keyword = 'connect'
+    JSONStore.objects.filter(view=view, key=keyword).all().delete()
+    data = json.loads(json.dumps(data, cls=EncodeAnything))
+    JSONStore.objects.create(
+        view=view,
+        key=keyword,
+        data=data,
+    )
+
+
+def create_jtbd_fund_cache():
+    print('create_jtbd_fund_cache')
+
+    # WalletConnect / ethers.js / TheDefiant
+        # Q(pk=275) | Q(pk=13) | Q(pk=567)
+    id_tuple= (275,13,567)
+    # id_tuple= (6,13,4)
+    projects = list(Grant.objects.filter(id__in=id_tuple).values(
+        'id', 'logo', 'title', 'admin_profile__handle',
+        'description', 'amount_received', 'amount_received_in_round', 'contributor_count',
+        'positive_round_contributor_count', 'in_active_clrs', 'clr_prediction_curve'
+    ).order_by('id', '-created_on').distinct('id'))
+    print(projects)
+    data = {
+        'projects': projects,
+        'builders': ['austintgriffith', 'alexmasmej', 'cryptomental', 'samczsun'],
+        'testimonial': {
+            'handle': 'sebastian',
+            'role': 'Python Developer',
+            'comment': "Transitioning to a career in crypto can be tough, but Gitcoin was a big help for me. Completing Gitcoin bounties and participating in Hackathons were invaluable for gaining exposure, experience, and of course making some money!",
+            'twitter': '',
+            'github': '',
+        },
+    }
+    view = 'jtbd'
+    keyword = 'fund'
+    JSONStore.objects.filter(view=view, key=keyword).all().delete()
+    data = json.loads(json.dumps(data, cls=EncodeAnything))
+    JSONStore.objects.create(
+        view=view,
+        key=keyword,
+        data=data,
+    )
```

Comment:
^ looks like page starting to grow in length !
Would it make sense to create 

`app/perftools/management/commands/cache/jtbd.py` and throw them there
We'll have to do it for the others as well 
we can import them here and use them in create_page_cache

Reply:
Agree, that is a good idea

Reply:
^ let's get this done as well before we merge @chibie / @octavioamu

Reply:
^ let's get this done as well before we merge @chibie / @octavioamu